tests/behavioural/features/steps/secure_messaging_context_helper.py

```diff
@@ -29,10 +29,10 @@ class SecureMessagingContextHelper:
     __DEFAULT_COLLECTION_EXERCISE = 'collection exercise1'
     __ALTERNATE_COLLECTION_EXERCISE = 'AnotherCollectionExercise'
 
-    __INTERNAL_USER_SPECIFIC_USER_ID = "SomeStringUntilWeGetIds"
+    __INTERNAL_USER_SPECIFIC_USER_ID = "SpecificInternalUserId"
     __INTERNAL_USER_GROUP = constants.NON_SPECIFIC_INTERNAL_USER
 
-    __ALTERNATIVE_INTERNAL_USER_SPECIFIC_USER_ID = "99951fcc-ed43-4cdb-ad1c-450f9986859b"
+    __ALTERNATIVE_INTERNAL_USER_SPECIFIC_USER_ID = "AlternateSpecificInternalUserId"
```

Comment:
I'm not a fan of this change.  I'd be interested in hearing peoples opinions but I look at tests sometimes to know what the inputs/outputs of a thing are, and what those inputs/outputs should roughly look like.  
I'd rather see a bunch of uuids for this and know that these ids should be uuids rather than some other random format.

Reply:
In the case of testing conversations having an indication of type in the db during tests makes tracking what is happening a lot easier. The data type used in the DB is var char not uuid . If the type was uuid then I would agree.

Reply:
Is it a varchar that always contains a uuid in preprod/prod?  or is it truly meant to contain whatever you want?

The question is mainly, was the db type NOT being uuid intentional?  If yes, then it's fine, but if not, then I don't think 'anything goes in the db' is a good enough reason.
As always, it's just an opinion, if you really want to leave it then I won't hold up this PR on something so minor ^^

Reply:
Spoke about this, while I still think it should be changed, I'm not going to hold up the PR on something so minor.